cdap-docs/developers-manual/source/building-blocks/index.rst

```diff
@@ -24,6 +24,7 @@ Building Blocks
     Spark Programs <spark-programs>
     Workers <workers>
     Workflows <workflows>
+    Artifacts <artifacts>
```

Comment:
Are you planning on adding an artifacts.rst file? I don't see it in this PR, and so this will not currently build.
This is the toctree, and lists all the other files in the directory to be built.